index.html

```diff
@@ -2,45 +2,265 @@
+        <div class="tags">
+          <div class="ruby">
+            <p class="font">Ruby on rails</p>
+          </div>
+          <div class="css">
+            <p class="font">css</p>
+          </div>
+          <div class="js">
+            <p class="font">JavScript</p>
+          </div>
+          <div class="html">
+            <p class="font">html</p>
+          </div>
         </div>
-        <div class="space"></div>
-      </section>
-    </main>
+        <div class="button-space">
+          <button type="button" class="project-text">See Project
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="block">
+      <div class="gray-area"></div>
+      <div class="description">
+        <div class="post">Multi-Post Stories Gain+Glory</div>
+        <div class="tags">
+          <div class="ruby">
+            <p class="font">Ruby on rails</p>
+          </div>
+          <div class="css">
+            <p class="font">css</p>
+          </div>
+          <div class="js">
+            <p class="font">JavScript</p>
+          </div>
+          <div class="html">
+            <p class="font">html</p>
+          </div>
+        </div>
+        <div class="button-space">
+          <button type="button" class="project-text">See Project
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="block">
+      <div class="gray-area"></div>
+      <div class="description">
+        <div class="post">Multi-Post Stories Gain+Glory</div>
+        <div class="tags">
+          <div class="ruby">
+            <p class="font">Ruby on rails</p>
+          </div>
+          <div class="css">
+            <p class="font">css</p>
+          </div>
+          <div class="js">
+            <p class="font">JavScript</p>
+          </div>
+          <div class="html">
+            <p class="font">html</p>
+          </div>
+        </div>
+        <div class="button-space">
+          <button type="button" class="project-text">See Project
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="block">
+      <div class="gray-area"></div>
+      <div class="description">
+        <div class="post">Multi-Post Stories Gain+Glory</div>
+        <div class="tags">
+          <div class="ruby">
+            <p class="font">Ruby on rails</p>
+          </div>
+          <div class="css">
+            <p class="font">css</p>
+          </div>
+          <div class="js">
+            <p class="font">JavScript</p>
+          </div>
+          <div class="html">
+            <p class="font">html</p>
+          </div>
+        </div>
+        <div class="button-space">
+          <button type="button" class="project-text">See Project
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="block">
+      <div class="gray-area"></div>
+      <div class="description">
+        <div class="post">Multi-Post Stories Gain+Glory</div>
```

Comment:
- Please, consider using an h2-h6 tag here, as this is the header and not a div item.